cog/core/downtime.py

```
@@ -0,0 +1,97 @@
+    if before is None:
+        before = datetime.now()
+
+    channel: Messageable = bot.get_channel(channel_id)
+
+    if not channel:
+        raise ValueError(f"Cannot get channel (id={channel}).")
+
+    if not isinstance(channel, Messageable):
+        raise ValueError(
+            f"{channel.name} (id={channel_id}, type={type(channel)}) is not messageable."
+        )
```

Ruff (0.8.2)

Prefer `TypeError` exception for invalid type

(TRY004)